CONTRIBUTING.md

```diff
@@ -418,6 +418,9 @@ Be sure to post in the PR conversation that you have made the requested changes.
 
 ### Other resources
 
+-   [Challenge Style Guide to create a new freeCodeCamp
```

Comment:
Perhaps mention something about "or to keep in mind when updating existing ones." Or maybe that's too wordy or already implied by what's said already?